src/collector.ts

```diff
@@ -412,7 +416,6 @@ const startCollector = async () => {
 
   // Initialize the database
   Storage.initializeDB()
-  Storage.addExitListeners(ws)
```

Comment:
This was not needed. Its meant for WS mode. MQ mode will use addSigListeners method only.